org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/real/ExampleMalcolmDeviceTest.java

```diff
@@ -54,8 +55,9 @@
 
 public class ExampleMalcolmDeviceTest {
 
-	private IMalcolmService      service;
+	private IScanService service;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "service" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)